lib/config/prettier.js

```diff
@@ -24,4 +24,21 @@ module.exports = {
     // https://github.com/prettier/eslint-config-prettier/blob/5399175c37466747aae9d407021dffec2c169c8b/README.md#no-unexpected-multiline
     'no-unexpected-multiline': 'error',
   }),
+
+  overrides: [
+    {
+      // force babylon parser for graphql files
```

Comment:
This comment is doesn't reflect the code.

eslint-plugin-prettier v2.7.0 forces the babylon parser for graphql files internally (big kudos to their maintainer for the rapid release of my PR), so we no longer need to explicitly turn that off to stop things from crashing.

You're right that we would run prettier over these files and no errors would be reported thanks to eslint-plugin-graphql [suppressing output for rules other than its own](https://github.com/apollographql/eslint-plugin-graphql/blob/fe6f6d7627605d2b1259bf19cba6985bf53fd671/src/index.js#L515-L518) (per our convo yesterday). I'm torn between having extra config to save a slight bit of time and having no config to be ready for a future where eslint-plugin-graphql exposes other rule violations and supports autofixing.

Reply:
my vote is to disable for now and re-enable when support is added. That way things don't magically break when a plugin unrelated to this config is updated.